label_studio/io_storages/aperturedb/api.py

```diff
@@ -0,0 +1,188 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+from django.utils.decorators import method_decorator
+from drf_yasg import openapi as openapi
+from drf_yasg.utils import swagger_auto_schema
+from io_storages.api import (
+    ExportStorageDetailAPI,
+    ExportStorageFormLayoutAPI,
+    ExportStorageListAPI,
+    ExportStorageSyncAPI,
+    ExportStorageValidateAPI,
+    ImportStorageDetailAPI,
+    ImportStorageFormLayoutAPI,
+    ImportStorageListAPI,
+    ImportStorageSyncAPI,
+    ImportStorageValidateAPI,
+)
+from io_storages.aperturedb.models import ApertureDBExportStorage, ApertureDBImportStorage
+from io_storages.aperturedb.serializers import ApertureDBExportStorageSerializer, ApertureDBImportStorageSerializer
+
+
+@method_decorator(
```

Comment:
Just for anyone looking in the future - this is called method_decorator, but you decorate a python class with the HTTP method ( mildly confusing )